Handlers/ReceiptsHandler.fs

```diff
@@ -0,0 +1,21 @@
+namespace ExpenseReports.Handlers
+
+open System.IO
+open Microsoft.AspNetCore.Http
+open Giraffe.HttpHandlers
+open Giraffe.Tasks
+
+module ReceiptHandler =
+    let receiptUploadHandler =
+        fun (next : HttpFunc) (ctx : HttpContext) ->
+            task {
```

Comment:
Is it necessary to wrap the whole thing in `task` or is it possible to only localize the `task` to something that is explicitly async?